### PR TITLE
update samples for june release

### PR DESCRIPTION
**Pull request by zhouwangzw**
## Purpose

* ...

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: Windows/csharp_samples/csharp_samples.csproj [hunk 1]
```diff
@@ -78,11 +79,11 @@
...
   <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
     <PropertyGroup>
       <ErrorText>This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is {0}.</ErrorText>
     </PropertyGroup>
-    <Error Condition="!Exists('packages\Microsoft.CognitiveServices.Speech.0.2.12733\build\net45\Microsoft.CognitiveServices.Speech.targets')" Text="$([System.String]::Format('$(ErrorText)', 'packages\Microsoft.CognitiveServices.Speech.0.2.12733\build\net45\Microsoft.CognitiveServices.Speech.targets'))" />
+    <Error Condition="!Exists('packages\Microsoft.CognitiveServices.Speech.0.4.0\build\net45\Microsoft.CognitiveServices.Speech.targets')" Text="$([System.String]::Format('$(ErrorText)', 'packages\Microsoft.CognitiveServices.Speech.0.4.0\build\net45\Microsoft.CognitiveServices.Speech.targets'))" />
```
Comment (wolfma61) [inline Windows/csharp_samples/csharp_samples.csproj]: >.0.4.0 [](start = 74, length = 6)

is this actually what will be the release nuget??

Comment (zhouwangzw) [reply Windows/csharp_samples/csharp_samples.csproj]: yes

File: Windows/Translation-WPF/samples/TranslationServiceExample/app.config [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <startup>
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1"/>
+  </startup>
+  <appSettings>
+    <add key="primaryKey" value="yourPrimaryKey"/>
+    <add key="luisAppID" value="yourLuisAppID"/>
+    <add key="luisSubscriptionID" value="yourLuisSubsrciptionID"/>
```
Comment (mahilleb-msft) [inline Windows/Translation-WPF/samples/TranslationServiceExample/app.config]: >yourLuisSubsrciptionID [](start = 41, length = 22)

fix typo

File: Windows/Translation-WPF/samples/TranslationServiceExample/App.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+// included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED ""AS IS"", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+// </copyright>
```
Comment (mahilleb-msft) [inline Windows/Translation-WPF/samples/TranslationServiceExample/App.xaml.cs]: remove

Comment (mahilleb-msft) [reply Windows/Translation-WPF/samples/TranslationServiceExample/App.xaml.cs]: (check elsewhere, too)

---
In reply to: [193678571](https://github.com/Azure-Samples/cognitive-services-speech-sdk/pull/7#discussion_r193678571) [](ancestors = 193678571)

File: Windows/Translation-WPF/samples/TranslationServiceExample/app.config (hunk 2 above)
Comment (mahilleb-msft) [reply Windows/Translation-WPF/samples/TranslationServiceExample/app.config]: (also elsewhere)

---
In reply to: [193678261](https://github.com/Azure-Samples/cognitive-services-speech-sdk/pull/7#discussion_r193678261) [](ancestors = 193678261)

File: Windows/csharp_samples/csharp_samples.csproj (hunk 1 above)
Comment (zhouwangzw) [reply Windows/csharp_samples/csharp_samples.csproj]: yes

---
In reply to: [193502396](https://github.com/Azure-Samples/cognitive-services-speech-sdk/pull/7#discussion_r193502396) [](ancestors = 193502396)

File: Windows/Translation-WPF/samples/TranslationServiceExample/app.config (hunk 2 above)
Comment (zhouwangzw) [reply Windows/Translation-WPF/samples/TranslationServiceExample/app.config]: it is not used. Removed.

---
In reply to: [193680359](https://github.com/Azure-Samples/cognitive-services-speech-sdk/pull/7#discussion_r193680359) [](ancestors = 193680359,193678261)

**Pull request merged**